package.json

```diff
@@ -67,10 +67,9 @@
     "postcss-loader": "^0.7.0",
     "postcss-nested": "^1.0.0",
     "postcss-simple-vars": "^1.1.0",
-    "react": "^15.6.2",
-    "react-addons-css-transition-group": ">=15.0.0",
-    "react-dom": "^15.6.2",
-    "react-test-renderer": "^15.6.2",
+    "react": "^16.6.3",
```

Comment:
Maybe we should change the version dependency to ">=16.0.0"

Reply:
I'm wondering if we need to limit it to <17 as well to protect ourselves

Reply:
Agreed

Reply:
">=16.0.0 <17.0.0"